README.md

```diff
@@ -22,19 +22,22 @@ panoptes is a service that can be used by:
 
 ### Option 1: Install via pypi and run server
```

Comment:
I think it's better if we leave option 1 as it is (for pypi) and we add the conda installation in a section named Option 2. Most people still confuse virtual environmnets and conda environments, so I thinks it's better if we also keep things separate.